Source/WebCore/accessibility/AXCoreObject.h

```diff
@@ -678,6 +678,25 @@ struct TextUnderElementMode {
     bool isHidden() { return considerHiddenState && inHiddenSubtree; }
 };
 
+struct LineRange {
+    unsigned startLineIndex;
+    unsigned endLineIndex;
```

Comment:
For any range data type I think it's helpful to include a comment saying whether the end index is inclusive or exclusive

Reply:
Good idea! Done in latest commit.